[iOS 18] Build on the public SDK

emw-apple opened the pull request:
#### 0a4bddf5cc108feed39f67bef24c7e5bcf9132fc
<pre>
[iOS 18] Build on the public SDK
<a href="https://rdar.apple.com/132780457">rdar://132780457</a>
<a href="https://bugs.webkit.org/show_bug.cgi?id=277325">https://bugs.webkit.org/show_bug.cgi?id=277325</a>

Reviewed by Alexey Proskuryakov.

Add TBD stubs generated using extract-tbds-from-internal-sdk.

Copy over SDK settings and AvailabilityProhibitedInternal.h, and remove
some symlinked headers that no longer need to be modified.

* WebKitLibraries/SDKs/iphoneos16.0-additions.sdk/System/Library/PrivateFrameworks/InstallCoordination.framework/InstallCoordination.tbd: Removed.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/SDKSettings.plist: Renamed from WebKitLibraries/SDKs/iphoneos16.0-additions.sdk/SDKSettings.plist.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/SymlinkedHeaders-output.xcfilelist: Renamed from WebKitLibraries/SDKs/iphoneos16.0-additions.sdk/SymlinkedHeaders-output.xcfilelist.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/SymlinkedHeaders.xcfilelist: Renamed from WebKitLibraries/SDKs/iphoneos16.0-additions.sdk/SymlinkedHeaders.xcfilelist.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/AppServerSupport.framework/AppServerSupport.tbd: Renamed from WebKitLibraries/SDKs/iphoneos16.0-additions.sdk/System/Library/PrivateFrameworks/AppServerSupport.framework/AppServerSupport.tbd.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/AppStoreDaemon.framework/AppStoreDaemon.tbd: Renamed from WebKitLibraries/SDKs/iphoneos16.0-additions.sdk/System/Library/PrivateFrameworks/AppStoreDaemon.framework/AppStoreDaemon.tbd.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/AppSupport.framework/AppSupport.tbd: Renamed from WebKitLibraries/SDKs/iphoneos16.0-additions.sdk/System/Library/PrivateFrameworks/AppSupport.framework/AppSupport.tbd.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/ApplePushService.framework/ApplePushService.tbd: Renamed from WebKitLibraries/SDKs/iphoneos16.0-additions.sdk/System/Library/PrivateFrameworks/ApplePushService.framework/ApplePushService.tbd.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/AuthKit.framework/AuthKit.tbd: Copied from WebKitLibraries/SDKs/iphoneos16.0-additions.sdk/System/Library/PrivateFrameworks/AuthKit.framework/AuthKit.tbd.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/BackBoardServices.framework/BackBoardServices.tbd: Renamed from WebKitLibraries/SDKs/iphoneos16.0-additions.sdk/System/Library/PrivateFrameworks/BackBoardServices.framework/BackBoardServices.tbd.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/CoreAnalytics.framework/CoreAnalytics.tbd: Copied from WebKitLibraries/SDKs/iphoneos16.0-additions.sdk/System/Library/PrivateFrameworks/AuthKit.framework/AuthKit.tbd.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/CorePrediction.framework/CorePrediction.tbd: Renamed from WebKitLibraries/SDKs/iphoneos16.0-additions.sdk/System/Library/PrivateFrameworks/CorePrediction.framework/CorePrediction.tbd.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/FontServices.framework/libFontParser.tbd: Renamed from WebKitLibraries/SDKs/iphoneos16.0-additions.sdk/System/Library/PrivateFrameworks/FontServices.framework/libFontParser.tbd.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/FrontBoardServices.framework/FrontBoardServices.tbd: Renamed from WebKitLibraries/SDKs/iphoneos16.0-additions.sdk/System/Library/PrivateFrameworks/FrontBoardServices.framework/FrontBoardServices.tbd.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/GraphicsServices.framework/GraphicsServices.tbd: Renamed from WebKitLibraries/SDKs/iphoneos16.0-additions.sdk/System/Library/PrivateFrameworks/GraphicsServices.framework/GraphicsServices.tbd.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/IOSurfaceAccelerator.framework/IOSurfaceAccelerator.tbd: Renamed from WebKitLibraries/SDKs/iphoneos16.0-additions.sdk/System/Library/PrivateFrameworks/IOSurfaceAccelerator.framework/IOSurfaceAccelerator.tbd.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/IconServices.framework/IconServices.tbd: Added.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/IdleTimerServices.framework/IdleTimerServices.tbd: Renamed from WebKitLibraries/SDKs/iphoneos16.0-additions.sdk/System/Library/PrivateFrameworks/AuthKit.framework/AuthKit.tbd.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/InstallCoordination.framework/InstallCoordination.tbd: Copied from WebKitLibraries/SDKs/iphoneos16.0-additions.sdk/System/Library/PrivateFrameworks/VisionKitCore.framework/VisionKitCore.tbd.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/MobileKeyBag.framework/MobileKeyBag.tbd: Added.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/PrototypeTools.framework/PrototypeTools.tbd: Renamed from WebKitLibraries/SDKs/iphoneos16.0-additions.sdk/System/Library/PrivateFrameworks/PrototypeTools.framework/PrototypeTools.tbd.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/RemotePairingDevice.framework/RemotePairingDevice.tbd: Added.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/RunningBoardServices.framework/RunningBoardServices.tbd: Renamed from WebKitLibraries/SDKs/iphoneos16.0-additions.sdk/System/Library/PrivateFrameworks/RunningBoardServices.framework/RunningBoardServices.tbd.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/SafariSafeBrowsing.framework/SafariSafeBrowsing.tbd: Renamed from WebKitLibraries/SDKs/iphoneos16.0-additions.sdk/System/Library/PrivateFrameworks/SafariSafeBrowsing.framework/SafariSafeBrowsing.tbd.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/SharedWebCredentials.framework/SharedWebCredentials.tbd: Added.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/UIIntelligenceSupport.framework/UIIntelligenceSupport.tbd: Added.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/URLFormatting.framework/URLFormatting.tbd: Renamed from WebKitLibraries/SDKs/iphoneos16.0-additions.sdk/System/Library/PrivateFrameworks/URLFormatting.framework/URLFormatting.tbd.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/VisionKitCore.framework/VisionKitCore.tbd: Renamed from WebKitLibraries/SDKs/iphoneos16.0-additions.sdk/System/Library/PrivateFrameworks/VisionKitCore.framework/VisionKitCore.tbd.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/WritingTools.framework/WritingTools.tbd: Added.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/WritingToolsUI.framework/WritingToolsUI.tbd: Added.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/usr/local/include/AvailabilityProhibitedInternal.h: Copied from WebKitLibraries/SDKs/iphoneos16.0-additions.sdk/usr/local/include/AvailabilityProhibitedInternal.h.
* WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/Frameworks/AudioToolbox.framework/AudioToolbox.tbd: Removed.
* WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/Frameworks/CFNetwork.framework/CFNetwork.tbd: Removed.
* WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/Frameworks/IOKit.framework/IOKit.tbd: Removed.
* WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/Frameworks/IOKit.framework/Versions/A/IOKit.tbd: Removed.
* WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/Frameworks/IOKit.framework/Versions/Current: Removed.
* WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/Frameworks/IOSurface.framework/IOSurface.tbd: Removed.
* WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/Frameworks/Metal.framework/Metal.tbd: Removed.
* WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/Frameworks/QuartzCore.framework/QuartzCore.tbd: Removed.
* WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/PrivateFrameworks/InstallCoordination.framework/InstallCoordination.tbd: Removed.
* WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/usr: Removed.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/SDKSettings.plist: Renamed from WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/SDKSettings.plist.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/SymlinkedHeaders-output.xcfilelist: Renamed from WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/SymlinkedHeaders-output.xcfilelist.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/SymlinkedHeaders.xcfilelist: Renamed from WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/SymlinkedHeaders.xcfilelist.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/System/Library/PrivateFrameworks/AppServerSupport.framework/AppServerSupport.tbd: Renamed from WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/PrivateFrameworks/AppServerSupport.framework/AppServerSupport.tbd.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/System/Library/PrivateFrameworks/AppStoreDaemon.framework/AppStoreDaemon.tbd: Renamed from WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/PrivateFrameworks/AppStoreDaemon.framework/AppStoreDaemon.tbd.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/System/Library/PrivateFrameworks/AppSupport.framework/AppSupport.tbd: Copied from WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/PrivateFrameworks/AppSupport.framework/AppSupport.tbd.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/System/Library/PrivateFrameworks/ApplePushService.framework/ApplePushService.tbd: Renamed from WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/PrivateFrameworks/ApplePushService.framework/ApplePushService.tbd.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/System/Library/PrivateFrameworks/AuthKit.framework/AuthKit.tbd: Renamed from WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/PrivateFrameworks/AuthKit.framework/AuthKit.tbd.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/System/Library/PrivateFrameworks/BackBoardServices.framework/BackBoardServices.tbd: Renamed from WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/PrivateFrameworks/BackBoardServices.framework/BackBoardServices.tbd.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/System/Library/PrivateFrameworks/CoreAnalytics.framework/CoreAnalytics.tbd: Renamed from WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/Frameworks/OpenGLES.framework/OpenGLES.tbd.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/System/Library/PrivateFrameworks/CorePrediction.framework/CorePrediction.tbd: Renamed from WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/PrivateFrameworks/CorePrediction.framework/CorePrediction.tbd.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/System/Library/PrivateFrameworks/FontServices.framework/libFontParser.tbd: Renamed from WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/PrivateFrameworks/FontServices.framework/libFontParser.tbd.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/System/Library/PrivateFrameworks/FrontBoardServices.framework/FrontBoardServices.tbd: Renamed from WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/PrivateFrameworks/FrontBoardServices.framework/FrontBoardServices.tbd.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/System/Library/PrivateFrameworks/GraphicsServices.framework/GraphicsServices.tbd: Renamed from WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/PrivateFrameworks/GraphicsServices.framework/GraphicsServices.tbd.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/System/Library/PrivateFrameworks/IconServices.framework/IconServices.tbd: Renamed from WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/PrivateFrameworks/AppSupport.framework/AppSupport.tbd.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/System/Library/PrivateFrameworks/IdleTimerServices.framework/IdleTimerServices.tbd: Copied from WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/PrivateFrameworks/VisionKitCore.framework/VisionKitCore.tbd.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/System/Library/PrivateFrameworks/InstallCoordination.framework/InstallCoordination.tbd: Copied from WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/PrivateFrameworks/VisionKitCore.framework/VisionKitCore.tbd.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/System/Library/PrivateFrameworks/PrototypeTools.framework/PrototypeTools.tbd: Renamed from WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/PrivateFrameworks/PrototypeTools.framework/PrototypeTools.tbd.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/System/Library/PrivateFrameworks/RunningBoardServices.framework/RunningBoardServices.tbd: Renamed from WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/PrivateFrameworks/RunningBoardServices.framework/RunningBoardServices.tbd.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/System/Library/PrivateFrameworks/SafariSafeBrowsing.framework/SafariSafeBrowsing.tbd: Renamed from WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/PrivateFrameworks/SafariSafeBrowsing.framework/SafariSafeBrowsing.tbd.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/System/Library/PrivateFrameworks/SharedWebCredentials.framework/SharedWebCredentials.tbd: Added.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/System/Library/PrivateFrameworks/UIIntelligenceSupport.framework/UIIntelligenceSupport.tbd: Added.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/System/Library/PrivateFrameworks/URLFormatting.framework/URLFormatting.tbd: Renamed from WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/PrivateFrameworks/URLFormatting.framework/URLFormatting.tbd.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/System/Library/PrivateFrameworks/VisionKitCore.framework/VisionKitCore.tbd: Renamed from WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/PrivateFrameworks/VisionKitCore.framework/VisionKitCore.tbd.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/System/Library/PrivateFrameworks/WritingTools.framework/WritingTools.tbd: Added.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/usr/local/include/AvailabilityProhibitedInternal.h: Renamed from WebKitLibraries/SDKs/iphoneos16.0-additions.sdk/usr/local/include/AvailabilityProhibitedInternal.h.

Canonical link: <a href="https://commits.webkit.org/281605@main">https://commits.webkit.org/281605@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/3a8710ca81a8fb9a55e72eba819de7c4bf929848

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/60447 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/39798 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/13005 "Built successfully") | [  ~~🛠 wpe~~](https://ews-build.webkit.org/#/builders/5/builds/64370 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 wincairo~~](https://ews-build.webkit.org/#/builders/59/builds/10978 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/62577 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/47472 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/11211 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/5/builds/64370 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 wincairo-tests~~](https://ews-build.webkit.org/#/builders/59/builds/10978 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/62478 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/37074 "Passed tests") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/18/builds/52356 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-wpe~~](https://ews-build.webkit.org/#/builders/5/builds/64370 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/42/builds/33783 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/9585 "Passed tests") | [  ~~🛠 wpe-cairo~~](https://ews-build.webkit.org/#/builders/65/builds/9899 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/13/builds/55655 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/9875 "Passed tests") | [  ~~🛠 gtk~~](https://ews-build.webkit.org/#/builders/2/builds/66098 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/87/builds/4380 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/62/builds/9710 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/2/builds/66098 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/86/builds/4399 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/52336 "Passed tests") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/2/builds/66098 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/13432 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/88/builds/3639 "Passed tests") | | | 
| | [  ~~🛠 tv~~](https://ews-build.webkit.org/#/builders/44/builds/35610 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
| | [  ~~🛠 tv-sim~~](https://ews-build.webkit.org/#/builders/46/builds/36692 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
| | [  ~~🛠 watch~~](https://ews-build.webkit.org/#/builders/43/builds/37783 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
| | [  ~~🛠 watch-sim~~](https://ews-build.webkit.org/#/builders/45/builds/36434 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
<!--EWS-Status-Bubble-End-->

aproskuryakov reviewed (approved):
I wonder if there is a way to convince git that these are new files, not copies of random old files (which it often gets wrong).

emw-apple committed:
[iOS 18] Build on the public SDK
rdar://132780457
https://bugs.webkit.org/show_bug.cgi?id=277325

Reviewed by Alexey Proskuryakov.

Add TBD stubs generated using extract-tbds-from-internal-sdk.

Copy over SDK settings and AvailabilityProhibitedInternal.h, and remove
some symlinked headers that no longer need to be modified.

* WebKitLibraries/SDKs/iphoneos16.0-additions.sdk/System/Library/PrivateFrameworks/InstallCoordination.framework/InstallCoordination.tbd: Removed.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/SDKSettings.plist: Renamed from WebKitLibraries/SDKs/iphoneos16.0-additions.sdk/SDKSettings.plist.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/SymlinkedHeaders-output.xcfilelist: Renamed from WebKitLibraries/SDKs/iphoneos16.0-additions.sdk/SymlinkedHeaders-output.xcfilelist.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/SymlinkedHeaders.xcfilelist: Renamed from WebKitLibraries/SDKs/iphoneos16.0-additions.sdk/SymlinkedHeaders.xcfilelist.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/AppServerSupport.framework/AppServerSupport.tbd: Renamed from WebKitLibraries/SDKs/iphoneos16.0-additions.sdk/System/Library/PrivateFrameworks/AppServerSupport.framework/AppServerSupport.tbd.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/AppStoreDaemon.framework/AppStoreDaemon.tbd: Renamed from WebKitLibraries/SDKs/iphoneos16.0-additions.sdk/System/Library/PrivateFrameworks/AppStoreDaemon.framework/AppStoreDaemon.tbd.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/AppSupport.framework/AppSupport.tbd: Renamed from WebKitLibraries/SDKs/iphoneos16.0-additions.sdk/System/Library/PrivateFrameworks/AppSupport.framework/AppSupport.tbd.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/ApplePushService.framework/ApplePushService.tbd: Renamed from WebKitLibraries/SDKs/iphoneos16.0-additions.sdk/System/Library/PrivateFrameworks/ApplePushService.framework/ApplePushService.tbd.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/AuthKit.framework/AuthKit.tbd: Copied from WebKitLibraries/SDKs/iphoneos16.0-additions.sdk/System/Library/PrivateFrameworks/AuthKit.framework/AuthKit.tbd.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/BackBoardServices.framework/BackBoardServices.tbd: Renamed from WebKitLibraries/SDKs/iphoneos16.0-additions.sdk/System/Library/PrivateFrameworks/BackBoardServices.framework/BackBoardServices.tbd.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/CoreAnalytics.framework/CoreAnalytics.tbd: Copied from WebKitLibraries/SDKs/iphoneos16.0-additions.sdk/System/Library/PrivateFrameworks/AuthKit.framework/AuthKit.tbd.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/CorePrediction.framework/CorePrediction.tbd: Renamed from WebKitLibraries/SDKs/iphoneos16.0-additions.sdk/System/Library/PrivateFrameworks/CorePrediction.framework/CorePrediction.tbd.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/FontServices.framework/libFontParser.tbd: Renamed from WebKitLibraries/SDKs/iphoneos16.0-additions.sdk/System/Library/PrivateFrameworks/FontServices.framework/libFontParser.tbd.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/FrontBoardServices.framework/FrontBoardServices.tbd: Renamed from WebKitLibraries/SDKs/iphoneos16.0-additions.sdk/System/Library/PrivateFrameworks/FrontBoardServices.framework/FrontBoardServices.tbd.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/GraphicsServices.framework/GraphicsServices.tbd: Renamed from WebKitLibraries/SDKs/iphoneos16.0-additions.sdk/System/Library/PrivateFrameworks/GraphicsServices.framework/GraphicsServices.tbd.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/IOSurfaceAccelerator.framework/IOSurfaceAccelerator.tbd: Renamed from WebKitLibraries/SDKs/iphoneos16.0-additions.sdk/System/Library/PrivateFrameworks/IOSurfaceAccelerator.framework/IOSurfaceAccelerator.tbd.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/IconServices.framework/IconServices.tbd: Added.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/IdleTimerServices.framework/IdleTimerServices.tbd: Renamed from WebKitLibraries/SDKs/iphoneos16.0-additions.sdk/System/Library/PrivateFrameworks/AuthKit.framework/AuthKit.tbd.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/InstallCoordination.framework/InstallCoordination.tbd: Copied from WebKitLibraries/SDKs/iphoneos16.0-additions.sdk/System/Library/PrivateFrameworks/VisionKitCore.framework/VisionKitCore.tbd.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/MobileKeyBag.framework/MobileKeyBag.tbd: Added.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/PrototypeTools.framework/PrototypeTools.tbd: Renamed from WebKitLibraries/SDKs/iphoneos16.0-additions.sdk/System/Library/PrivateFrameworks/PrototypeTools.framework/PrototypeTools.tbd.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/RemotePairingDevice.framework/RemotePairingDevice.tbd: Added.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/RunningBoardServices.framework/RunningBoardServices.tbd: Renamed from WebKitLibraries/SDKs/iphoneos16.0-additions.sdk/System/Library/PrivateFrameworks/RunningBoardServices.framework/RunningBoardServices.tbd.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/SafariSafeBrowsing.framework/SafariSafeBrowsing.tbd: Renamed from WebKitLibraries/SDKs/iphoneos16.0-additions.sdk/System/Library/PrivateFrameworks/SafariSafeBrowsing.framework/SafariSafeBrowsing.tbd.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/SharedWebCredentials.framework/SharedWebCredentials.tbd: Added.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/UIIntelligenceSupport.framework/UIIntelligenceSupport.tbd: Added.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/URLFormatting.framework/URLFormatting.tbd: Renamed from WebKitLibraries/SDKs/iphoneos16.0-additions.sdk/System/Library/PrivateFrameworks/URLFormatting.framework/URLFormatting.tbd.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/VisionKitCore.framework/VisionKitCore.tbd: Renamed from WebKitLibraries/SDKs/iphoneos16.0-additions.sdk/System/Library/PrivateFrameworks/VisionKitCore.framework/VisionKitCore.tbd.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/WritingTools.framework/WritingTools.tbd: Added.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/System/Library/PrivateFrameworks/WritingToolsUI.framework/WritingToolsUI.tbd: Added.
* WebKitLibraries/SDKs/iphoneos18.0-additions.sdk/usr/local/include/AvailabilityProhibitedInternal.h: Copied from WebKitLibraries/SDKs/iphoneos16.0-additions.sdk/usr/local/include/AvailabilityProhibitedInternal.h.
* WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/Frameworks/AudioToolbox.framework/AudioToolbox.tbd: Removed.
* WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/Frameworks/CFNetwork.framework/CFNetwork.tbd: Removed.
* WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/Frameworks/IOKit.framework/IOKit.tbd: Removed.
* WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/Frameworks/IOKit.framework/Versions/A/IOKit.tbd: Removed.
* WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/Frameworks/IOKit.framework/Versions/Current: Removed.
* WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/Frameworks/IOSurface.framework/IOSurface.tbd: Removed.
* WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/Frameworks/Metal.framework/Metal.tbd: Removed.
* WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/Frameworks/QuartzCore.framework/QuartzCore.tbd: Removed.
* WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/PrivateFrameworks/InstallCoordination.framework/InstallCoordination.tbd: Removed.
* WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/usr: Removed.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/SDKSettings.plist: Renamed from WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/SDKSettings.plist.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/SymlinkedHeaders-output.xcfilelist: Renamed from WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/SymlinkedHeaders-output.xcfilelist.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/SymlinkedHeaders.xcfilelist: Renamed from WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/SymlinkedHeaders.xcfilelist.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/System/Library/PrivateFrameworks/AppServerSupport.framework/AppServerSupport.tbd: Renamed from WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/PrivateFrameworks/AppServerSupport.framework/AppServerSupport.tbd.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/System/Library/PrivateFrameworks/AppStoreDaemon.framework/AppStoreDaemon.tbd: Renamed from WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/PrivateFrameworks/AppStoreDaemon.framework/AppStoreDaemon.tbd.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/System/Library/PrivateFrameworks/AppSupport.framework/AppSupport.tbd: Copied from WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/PrivateFrameworks/AppSupport.framework/AppSupport.tbd.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/System/Library/PrivateFrameworks/ApplePushService.framework/ApplePushService.tbd: Renamed from WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/PrivateFrameworks/ApplePushService.framework/ApplePushService.tbd.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/System/Library/PrivateFrameworks/AuthKit.framework/AuthKit.tbd: Renamed from WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/PrivateFrameworks/AuthKit.framework/AuthKit.tbd.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/System/Library/PrivateFrameworks/BackBoardServices.framework/BackBoardServices.tbd: Renamed from WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/PrivateFrameworks/BackBoardServices.framework/BackBoardServices.tbd.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/System/Library/PrivateFrameworks/CoreAnalytics.framework/CoreAnalytics.tbd: Renamed from WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/Frameworks/OpenGLES.framework/OpenGLES.tbd.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/System/Library/PrivateFrameworks/CorePrediction.framework/CorePrediction.tbd: Renamed from WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/PrivateFrameworks/CorePrediction.framework/CorePrediction.tbd.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/System/Library/PrivateFrameworks/FontServices.framework/libFontParser.tbd: Renamed from WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/PrivateFrameworks/FontServices.framework/libFontParser.tbd.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/System/Library/PrivateFrameworks/FrontBoardServices.framework/FrontBoardServices.tbd: Renamed from WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/PrivateFrameworks/FrontBoardServices.framework/FrontBoardServices.tbd.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/System/Library/PrivateFrameworks/GraphicsServices.framework/GraphicsServices.tbd: Renamed from WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/PrivateFrameworks/GraphicsServices.framework/GraphicsServices.tbd.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/System/Library/PrivateFrameworks/IconServices.framework/IconServices.tbd: Renamed from WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/PrivateFrameworks/AppSupport.framework/AppSupport.tbd.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/System/Library/PrivateFrameworks/IdleTimerServices.framework/IdleTimerServices.tbd: Copied from WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/PrivateFrameworks/VisionKitCore.framework/VisionKitCore.tbd.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/System/Library/PrivateFrameworks/InstallCoordination.framework/InstallCoordination.tbd: Copied from WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/PrivateFrameworks/VisionKitCore.framework/VisionKitCore.tbd.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/System/Library/PrivateFrameworks/PrototypeTools.framework/PrototypeTools.tbd: Renamed from WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/PrivateFrameworks/PrototypeTools.framework/PrototypeTools.tbd.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/System/Library/PrivateFrameworks/RunningBoardServices.framework/RunningBoardServices.tbd: Renamed from WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/PrivateFrameworks/RunningBoardServices.framework/RunningBoardServices.tbd.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/System/Library/PrivateFrameworks/SafariSafeBrowsing.framework/SafariSafeBrowsing.tbd: Renamed from WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/PrivateFrameworks/SafariSafeBrowsing.framework/SafariSafeBrowsing.tbd.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/System/Library/PrivateFrameworks/SharedWebCredentials.framework/SharedWebCredentials.tbd: Added.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/System/Library/PrivateFrameworks/UIIntelligenceSupport.framework/UIIntelligenceSupport.tbd: Added.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/System/Library/PrivateFrameworks/URLFormatting.framework/URLFormatting.tbd: Renamed from WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/PrivateFrameworks/URLFormatting.framework/URLFormatting.tbd.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/System/Library/PrivateFrameworks/VisionKitCore.framework/VisionKitCore.tbd: Renamed from WebKitLibraries/SDKs/iphonesimulator16.0-additions.sdk/System/Library/PrivateFrameworks/VisionKitCore.framework/VisionKitCore.tbd.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/System/Library/PrivateFrameworks/WritingTools.framework/WritingTools.tbd: Added.
* WebKitLibraries/SDKs/iphonesimulator18.0-additions.sdk/usr/local/include/AvailabilityProhibitedInternal.h: Renamed from WebKitLibraries/SDKs/iphoneos16.0-additions.sdk/usr/local/include/AvailabilityProhibitedInternal.h.

Canonical link: https://commits.webkit.org/281605@main
